charts/cinder-csi-plugin/values.yaml

```diff
@@ -67,6 +67,18 @@ csi:
         - operator: Exists
```

Comment:
needs version bump if any changes to chart.https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/cinder-csi-plugin/Chart.yaml

Reply:
done